src/python/pants/backend/python/rules/coverage.md

```diff
@@ -0,0 +1,27 @@
+## How Python Coverage Works
+- running pytest generates data and drops a sql file, `.coverage` in the chroot.
+- running `coverage combine` on a list of `.coverage` files merges them into a single `.coverage` sql file.
+- running `coverage html` or `coverage xml` on a single `.coverage` file generates a report.
+
+
+## Pants Architecture for Coverage
+- Leave specific coverage options (e.g XML vs. Json output) to each language through a subsystem for that language’s coverage provider, e.g. pytest-cov
```

Comment:
I guess that this is ok... assuming that the defaults are "mostly" in alignment, this might still mean that `./pants test --coverage $xlang $ylang` would generate something similarly shaped in `dist`.

Reply:
I was thinking the defaults would be `coverage/python`, `coverage/jvm`, etc.

Reply:
@Eric-Arellano : Yea, me too. I was mostly referring to the "XML vs JSON" part. Having those be similar would be good, although that will not always be possible.